src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.RegionBuilder.cs

```diff
@@ -231,6 +231,8 @@ public void ExtendToInclude(BasicBlockBuilder block)
                               Kind != ControlFlowRegionKind.TryAndFinally) ||
                               Regions.Last().LastBlock == block);
 
+                Debug.Assert(block != null);
```

Comment:
Minor point: I would expect this assert to be first since it is the simplest, especially since the more elaborate assert above will probably fail if `block == null`.

Reply:
> since the existing assert will probably fail if block == null

It does not, actually, in the specific case being tested here. I will move it first though.

---
In reply to: [239625068](https://github.com/dotnet/roslyn/pull/31592#discussion_r239625068) [](ancestors = 239625068)